### ISSUE TITLE
Blank texture in gg.visualization.drape_array_over_dem (+ default zmax)

### ISSUE THREAD

**Issue by spasquet** [cgre-aachen/gemgis#303]
Hi there,

I want to report an issue with `gg.visualization.drape_array_over_dem` that I managed to solve.
I was getting a texture filled with an array of ones, associated with a warning from `pv.numpy_to_texture` telling me my array was not in uint8.

It turns out that `skimage.transform.resize` was changing the format of the array (scikit-image 0.21.0 installed)
Adding option `preserve_range=True` to the `skimage.transform.resize` function solved it :

```
if resize_array:
            array = resize(image=array,
                           preserve_range=True,
                           output_shape=(dem.shape[0],
                                         dem.shape[1]))
        else:
            dem = resize(image=dem,
                         preserve_range=True,
                         output_shape=(array.shape[0],
                                       array.shape[1]))
```

Also, could you please change the default zmax in `gg.visualization.drape_array_over_dem` ?
It is now set to 1000...I was working with data in the Alps at 2500m and it took me forever to figure out why the draping wasn't functioning. There should only be a valid zmax when specifically mentioned. I have now set it to 10000 by default now, it does the trick.

Best
Sylvain

**Comment by AlexanderJuestel** [comment]
@spasquet Thank you for the raising this issue, I will have a look at it tomorrow first thing and it will be fixed in the next release!

**Comment by AlexanderJuestel** [comment]
Your fix was implemented in https://github.com/cgre-aachen/gemgis/commit/bfbd01e82f70cd9a4171349c289d2e5e50533e44. However, as we are changing the way we deploy the package to pypi, you can only download the Github repo to get the latest features. Sorry for this inconvenience

**Comment by spasquet** [comment]
Great thanks !

**Comment by AlexanderJuestel** [comment]
@spasquet I have released GemGIS 1.1.3 yesterday. You may want to check it out :)
